app/services/idv/steps/inherited_proofing/user_pii_managable.rb

```diff
@@ -14,6 +14,8 @@ def inherited_proofing_save_user_pii_to_session!
         def inherited_proofing_save_session!
           return unless inherited_proofing_form_response.success?
 
+          mark_step_complete(:verify_wait)
```

Comment:
I would suggest moving this out of this module because it binds it to the :verify_wait step. Perhaps to `VerifyWaitStepShow#process_async_state` (see comments).

Reply:
You know, I went back and forth on that exact thing. Thanks for the gut check. I'll make that change.